core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -5646,27 +5650,151 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
 #endif
                 goto call_func_from_entry;
             }
-#if WASM_ENABLE_SIMD != 0
+#if defined(WASM_ENABLE_SIMDE) && WASM_ENABLE_SIMDE != 0
```

Comment:
Suggest to add #ifndef WASM_ENABLE_SIMDE #define WASM_ENABLE_SIMDE 0 #endif in core/config.h, so as to make WASM_ENABLE_SIMDE always defined, and then change this line to `#if WASM_ENABLE_SIMDE != 0`